examples/medplum-provider/src/hooks/useEncounter.ts

```diff
@@ -0,0 +1,16 @@
+import { OperationOutcome, Encounter } from '@medplum/fhirtypes';
+import { useResource } from '@medplum/react';
+import { useParams } from 'react-router';
+
+type Options = {
+  ignoreMissingEncounterId?: boolean;
+  setOutcome?: (outcome: OperationOutcome) => void;
+};
+
+export function useEncounter(options?: Options): Encounter | undefined {
```

Comment:
It looks like we don't ever use the `options` argument at any of the call sites, so maybe we can remove it?